controllers/ingressnodefirewall_controller_rules_test.go

```diff
@@ -108,52 +108,6 @@ var _ = Describe("IngressNodeFirewall controller rules", func() {
 				},
 			},
 		},
-		"duplicate rules and interfaces shall throw an error due to duplicate order": {
```

Comment:
I don't see how the APIs will prevent this from happening?

This test tests what happens when:
* 2 different IngressNodeFirewall objects are created and must be merged into a single IngressNodeFirewallNodeState